### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by HeyItsJono** [MahdiGhiasi/Xpotify#124]
Just wondering if the project is dead or if development is still ongoing?

**Comment by d4f5409d** [comment]
It is dead, sadly.

**Comment by DieterKoblenz** [comment]
Sad. 100 times better than the rubbish spotify app.

**Comment by WolfganP** [comment]
Yep. Spotify's official web player seems to be capped on purpose... are there other alternatives?
